compiler/nim.nim

```diff
@@ -80,14 +80,25 @@ proc handleCmdLine(cache: IdentCache; conf: ConfigRef) =
     else:
       conf.projectPath = canonicalizePath(conf, getCurrentDir())
     loadConfigs(DefaultConfig, cache, conf) # load all config files
-    let scriptFile = conf.projectFull.changeFileExt("nims")
-    if fileExists(scriptFile):
-      runNimScript(cache, scriptFile, freshDefines=false, conf)
+
+    proc runNimScriptIfExists(scriptFile:string)=
+      if fileExists(scriptFile):
+        runNimScript(cache, scriptFile, freshDefines=false, conf)
+
+    # TODO:
+    # merge this complex logic with `loadConfigs`
+    # check whether these should be controlled via
+    # optSkipConfigFile, optSkipUserConfigFile
+    let config_nims = "config.nims"
```

Comment:
That's a ``const`` and the Nim compiler doesn't use under_scores.

Reply:
done